README.rst

```diff
@@ -107,6 +107,37 @@ Installing this package will install ``nvidia-dali`` and its dependencies, if th
 
 ----
 
+Compiling DALI from source (Docker) - recommended
+-------------------------------------------------
+
+Prerequisites
+^^^^^^^^^^^^^
+
+.. |docker link| replace:: **Docker**
+.. _docker link: https://docs.docker.com/install/
+
+.. table::
+   :align: center
+
+   +----------------------------------------+---------------------------------------------------------------------------------------------+
+   | Linux x64                              |                                                                                             |
+   +----------------------------------------+---------------------------------------------------------------------------------------------+
+   | |docker link|_                         | Follow installation guide and manual at the link.                                           |
+   +----------------------------------------+---------------------------------------------------------------------------------------------+
+
+Building Docker image
```

Comment:
Building **using** Docker image?

Reply:
Also what could be nice to have here is info, why is there a "docker build" in the first place (for me it's not really clear, what can we use it for except CI).
I might be wrong here, but IMHO it's not really prepared for development (unless sbdy has sth like CLion), training only for TF. Turns out it's mostly for building whl, which is I think way simpler using bash one-liner

Reply:
Not really. Caffe and MXNet provide you a way to build docker image with them inside. In case of DALI it will give you clean env where DALI is inside and you can install upstream FW. It will allow the user to check if his env is broken or DALI itself (in case of some incompatibility between libraries present in the system or something similar).

Reply:
OK, I get it.

It's just that it's not really "clean", when there's TF inside ;)